Magento_RequisitionList/templates/requisition/view/items/toolbar/massaction/copy.phtml

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
I talk with Ania about it, and if there's situation when we need to override template just to add 1,2 classes it's better to use `extend` for it or add styles in theme